networkx/algorithms/triads.py

```diff
@@ -54,7 +56,7 @@ def triadic_census(G):
     Returns
     -------
     census : dict
-       Dictionary with triad names as keys and number of occurrences as values.
+       Dictionary with triad type as keys and number of occurrences as values.
```

Comment:
@bakerwho I'm wondering if there might be a good opportunity for educating end-users using the docs by adding  a very brief summary of what the 16 possible types are? To be clear, I don't mean a complete elucidation of them, but more along the lines of the semantic meaning of each of those triad types? Providing this in the docs might be helpful for newcomers to network science, and make NetworkX an even better resource for learning.

Reply:
@ericmjl this is a great idea. As linked in the original PR, Tom Snijders's slides ([here](http://www.stats.ox.ac.uk/~snijders/Trans_Triads_ha.pdf)) on this are a great resource.

Here's a TLDR version:
There can be 6 unique edges in a triad (order-3 DiGraph) (so 2^^6=64 unique triads given 3 nodes). These 64 triads each display exactly 1 of 16 topologies of triads (topologies can be permuted). These topologies are identified by the following notation:
{m}{a}{n}{type} (for example: 111D, 210, 102), where
- {m} = number of mutual ties (takes 0, 1, 2, 3); a mutual tie is (0,1) AND (1,0)
- {a} = number of assymmetric ties (takes 0, 1, 2, 3); an assymmetric tie is (0,1) BUT NOT (1,0) or vice versa
- {n} = number of null ties (takes 0, 1, 2, 3); a null tie is NEITHER (0,1) NOR (1,0)
- {type} = a letter (takes U, D, C, T) corresponding to up, down, cyclical and transitive. This is only used for topologies that can have more than one form (eg: 021D and 021U). Usage is better understood from the following diagram (taken from Snijders' slides):

<img width="689" alt="Screen Shot 2019-12-07 at 6 01 27 PM" src="https://user-images.githubusercontent.com/16526730/70382106-bd5cac00-191b-11ea-82ee-be20e22608f4.png">

Reply:
Wonderful! I think the tl;dr you provided is a very nice and concise description. What do you think about adding it in, then? I propose we go for it! :smile:

Reply:
Sure, I'll add it to the docstring! Also, @ericmjl could you maybe take a look at the Appveyor and Travis reports and help me fix it? They're both failing. Thanks!

Reply:
Sure, I'll add it in the docstring!